Casks/g/garmin-express.rb

```diff
@@ -1,17 +1,30 @@
 cask "garmin-express" do
-  version "7.22.1.0,7220100"
-  sha256 :no_check
+  on_catalina :or_older do
+    version "7.18.0"
+    sha256 "f032349fa2b4cd5abe20ae9201c905874a4d3ef1b328966a29d24e6e15793729"
+
+    url "https://download.garmin.com/omt/express/GarminExpress_#{version}_OS10x.dmg"
+
+    livecheck do
+      skip "Legacy version"
+    end
+  end
+  on_big_sur :or_newer do
+    version "7.22.1"
+    sha256 :no_check
+
+    url "https://download.garmin.com/omt/express/GarminExpress.dmg"
+
+    livecheck do
+      url "https://support.garmin.com/capi/faq/details/?locale=en-US&faqId=9MuiEv9c2y2wgcXvzEVEe8"
+      regex(/for\s+Mac:.*?v?(\d+(?:\.\d+)+)/i)
+    end
```

Comment:
```suggestion
      strategy :json do |json, regex|
        match = json["content"]&.match(regex)
        next if match.blank?

        match[1]
      end
    end
```

This is JSON data and there are unrelated numeric versions in other parts of the data (e.g., `contentVersion`), so it probably wouldn't hurt to restrict matching to the `content` text that we're targeting.